examples/hybrid/define_post_processing.jl

```diff
@@ -370,3 +370,181 @@ function paperplots_baro_wave_ρe(sol, output_dir, p, nlat, nlon)
+            weightfile,
+            meshfile_cc,
+            meshfile_rll,
+            meshfile_overlap;
+            in_type = "cgll",
+            in_np = Nq,
+        )
+
+        datafile_latlon = output_dir * "/bw-remapped_day" * string(day) * ".nc"
+        apply_remap(
+            datafile_latlon,
+            datafile_cc,
+            weightfile,
+            ["pres", "T", "vort", "qt", "w", "rho", "cloud_water"],
+        )
+
+        rm(remap_tmpdir, recursive = true)
+    end
+
+    # create plots as in the reference
+    for day in days
+        datafile_latlon = output_dir * "/bw-remapped_day" * string(day) * ".nc"
+        ncdata = NCDataset(datafile_latlon, "r")
+        lon = ncdata["lon"][:]
+        lat = ncdata["lat"][:]
+
+        p = ncdata["pres"][:]
+        T = ncdata["T"][:]
+        vort = ncdata["vort"][:] * FT(1e5)
+        qt = ncdata["qt"][:] * FT(1e3)
+        cloud_water = ncdata["cloud_water"][:]*FT(1e3)
+        rho = ncdata["rho"][:]
+        w = ncdata["w"][:]
+
+        vert_intg_cloud_water = sum( cloud_water.*rho, dims = 3) ./ sum(rho, dims=3)
+
+        latidx = findall(x -> x >= 0, lat)
+        lonidx = findall(x -> 0 <= x <= 240, lon)
+
+        plot_p = contourf(
+            lon[lonidx],
+            lat[latidx],
+            p[lonidx, latidx, 1, 1]',
+            color = :rainbow,
+            title = "pressure (1500m) day " * string(day),
+        )
+        png(plot_p, output_dir * "/bw-pressure-day" * string(day) * ".png")
+
+        plot_T = contourf(
+            lon[lonidx],
+            lat[latidx],
+            T[lonidx, latidx, 1, 1]',
+            color = :rainbow,
+            levels = 220:10:310,
+            title = "temperature (1500m) day " * string(day),
+        )
+        png(plot_T, output_dir * "/bw-temperature-day" * string(day) * ".png")
+
+        plot_ω = contourf(
+            lon[lonidx],
+            lat[latidx],
+            vort[lonidx, latidx, 1, 1]',
+            color = :balance,
+            title = "vorticity (1500m) day " * string(day),
+        )
+        png(plot_ω, output_dir * "/bw-vorticity-day" * string(day) * ".png")
+
+        plot_qt = contourf(
+            lon[lonidx],
+            lat[latidx],
+            qt[lonidx, latidx, 1, 1]',
+            color = :balance,
+            title = "qt (1500m) day " * string(day),
+        )
+        png(plot_qt, output_dir * "/bw-qt-day" * string(day) * ".png")
+
+        plot_w = contourf(
+            lon[lonidx],
+            lat[latidx],
+            w[lonidx, latidx, 1, 1]',
+            color = :balance,
+            title = "w (1500m) day " * string(day),
+        )
+        png(plot_w, output_dir * "/bw-w-day" * string(day) * ".png")
+
+        plot_cw = contourf(
+            lon[lonidx],
+            lat[latidx],
+            vert_intg_cloud_water[lonidx, latidx, 1, 1]',
+            color = :balance,
+            title = "cloud water (vertically integrated) day " * string(day),
+        )
```

Comment:
I would maybe label it as cloud condensate. Otherwise it might be confusing if its plotting only liquid water or also cloud ice. I would also ad units to the title since we are plotting in [g/kg]